app.py

```diff
@@ -61,28 +63,31 @@ def get_bool_env(env_name, default=False):
     return True
 
 
-def update_cfg_by_env(cfg):
+def update_cfg_by_env(cfg: Dict[str, Any]) -> None:
     """Update the config with data from environment variables."""
     # Get some configuration parameters from env.
-    env_model_settings = {}
-    env_model_settings['region'] = os.getenv('S3_REGION')
-    env_model_settings['endpoint_url'] = os.getenv('S3_URL')
-    env_model_settings['bucket_name'] = os.getenv('S3_BUCKET')
-    env_model_settings['base_path'] = os.getenv('S3_BASE_PATH')
-    env_model_settings['access_key_id'] = os.getenv('S3_ACCESS_KEY')
-    env_model_settings['secret_access_key'] = os.getenv('S3_SECRET_KEY')
-    env_model_settings['public_read'] = get_bool_env('S3_PUBLIC_READ', False)
-    env_model_settings['force_sync'] = get_bool_env('RWS_FORCE_SYNC', False)
+    env_model_settings = {
+        'region': os.getenv('S3_REGION'),
+        'endpoint_url': os.getenv('S3_URL'),
+        'bucket_name': os.getenv('S3_BUCKET'),
+        'base_path': os.getenv('S3_BASE_PATH'),
+        'access_key_id': os.getenv('S3_ACCESS_KEY'),
+        'secret_access_key': os.getenv('S3_SECRET_KEY'),
+        'public_read': get_bool_env('S3_PUBLIC_READ', False),
+        'force_sync': get_bool_env('RWS_FORCE_SYNC', False)
+    }
     gpg_key_armored = os.getenv('GPG_SIGN_KEY_ARMORED')
     # GPG_SIGN_KEY_ARMORED stores GPG secret key for signing the repositories
     # metadata. Our task is to add this key to the system and get its name.
     if gpg_key_armored:
-        env_model_settings['gpg_sign_key'] = \
-            add_gpg_key(gpg_key_armored.encode('ascii'))
+        env_model_settings['gpg_sign_key'] = add_gpg_key(gpg_key_armored.encode('ascii'))
 
-    env_common_settings = {}
-    env_common_settings['credentials'] = \
-        json.loads(os.environ.get('RWS_CREDENTIALS'))
+    if (cred := os.environ.get('RWS_CREDENTIALS')) is None:
```

Comment:
be careful: this assignment is allowed from `python 3.8 `

Reply:
Not related with the commit.
given your comment, I would prefer it to stay the way it was.

Reply:
I don't think so due to in case of staying with your way I should describe type manually. It is a wrong way I guess, otherwise, the type would be assigned dynamically.